### ISSUE TITLE
Loading icons for everything

### ISSUE THREAD

**Issue by IRHM** [sbondCo/Watcharr#6]
Loading icons for switching pages.

There should be either a loading icon for poster images, or a better default background image that doesn't show the alt text.

Anything to make the loading of images look nicer too, maybe a fade in when they finally load.
